### DISCUSSION TITLE
Is there any reverse operation to ReadPixels?

### DISCUSSION THREAD

**Discussion by Cyl18** [dlemstra/Magick.NET#769] [Help] [answered]
I know I can convert a byte array to an image with 'ReadPixels', but how can I convert it back to a byte array?😀
(read it pixel by pixel is not so efficient)

**Comment by dlemstra** [comment, marked as answer]
You could use `image.GetPixels().ToByteArray` for that.

**Reply by Cyl18** [reply]
thank you!
